Tensile/KernelWriterAssembly.py

```diff
@@ -2640,9 +2644,14 @@ def functionSignature(self, kernel ):
             offsetIsVgpr = True
           else:
             offsetIsVgpr = False
+        elif indices[i] in kernel["ProblemType"]["MirrorDims%s" % tc]:
+          offsetVgpr = True
```

Comment:
Should `offsetVgpr = True` on line 2648 be `offsetIsVgpr = True` ?

Reply:
Not sure MirrorDim feature is tested extensively ; lets change line num: 2648 to offsetIsVgpr

Reply:
Thanks! Fixed in 1bd95a7

Currently one [test](https://github.com/vsrad/Tensile/blob/mirror-dims/Tensile/Tests/extended/mirror_dims/mirror_dims_1sum.yaml) is not passing. The error occurs in the assembly implementation when calculating `incrementSrd` using stagger loop. I'll try to fix it as soon as possible.